public/data/abstraction.json

```
@@ -9,13 +9,23 @@
       "components": [
         "marker-list",
         "layers-graph"
-      ]
+      ],
+      "actions": []
     },
     {
       "title": "Configureren",
       "components": [
         "marker-list",
         "marker-configuration"
+      ],
+      "actions": [
+        "reset-button"
+      ]
+    },
+    {
+      "actions": [
+        "reset-button",
+        "save-button"
```

The 'button' part in the items now seems a bit unnecessary.

```suggestion
        "reset",
        "save"
```

---

Changed!